src/shell.js

```diff
@@ -233,12 +233,13 @@ if (ENVIRONMENT_IS_SHELL) {
     };
   }
 
-  if (typeof console !== 'undefined') {
-    // Support odd shell environments that lack console.* but have other stuff.
+  if (typeof print !== 'undefined') {
```

Comment:
Is this really a good idea? There are other methods on `console` that perhaps shouldn't be removed.

Reply:
True, in browsers there is also the profiling stuff. I don't think any of that works in the shell environment which is what this code is changing. But yeah, we could be more careful - it would increase code size, though, which is why I'm unhappy about doing that.

Reply:
It's about 50 bytes more, I pushed a commit with that...